services/web-app/pages/api/auth-callback.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { getPassportInstance } from '@carbon-platform/api/auth'
+
+import requireSession from '../../middleware/requireSession'
+
+const authCallback = requireSession().get(
+  (await getPassportInstance()).authenticate('prepiam.ice.ibmcloud.com'),
```

Comment:
This hostname seems like something that should be internal to the auth api. I wonder if it'd be worthing having another helper in the auth package called like `getAuthenticatedPassportInstance` that does `getPassportInstance` along with `authenticate`